demo/photos/preact/photos.js

```diff
@@ -49,7 +49,10 @@ export class Sample extends BaseSample {
           totalItems: items.length,
           component,
           resetValue,
-          ref: c => this._root = c.base
+          ref: c => {
+            this._root = c.base;
+            this._root.style.cssText = document.body.style.cssText;
```

Comment:
Is this change related?

Reply:
not really - this fixes some styling issue in the react photos demo (we style the body in the base class, but the react-repeater creates a `div` wrapper where it renders the content, so here we copy the body styles to the actual element that needs those) 

I Should probably move this in a separate PR...

Reply:
No big deal, feel free to commit to master directly perhaps.